### PR TITLE
add DownloadClicked/DownloadCancelled and FeedLoadError events; post …

### PR DESCRIPTION
**Pull request by angitbit**
…them along w/ errors so autoupdater can relay them as stats

### CODE REVIEW

File: SparkleDotNET/SUConstants.cs [hunk 1]
```diff
@@ -13,10 +13,11 @@ public class SUConstants {
         public const string SUPublicRSAKeyFileKey = "SUPublicRSAKeyFile";
         public const string SUNoUpdateError = "com.Sparkle.NoUpdate";
         public const string SUNoFeedURLError = "com.Sparkle.NoFeedURL";
+        public const string SUFeedLoadError = "FeedLoadError";
         public const string SUSignatureError = "com.Sparkle.SignatureError";
         public const string SUNoInstallerError = "com.Sparkle.NoInstallerError";
         public const string SUInstallerFailedToLaunchError = "com.Sparkle.InstallerFailedToLaunch";
-        public const string SUDownloadFailedError = "com.Sparkle.DownloadFailedError";
```
Comment (angitbit) [inline SparkleDotNET/SUConstants.cs:19]: unused, seems covered by AbortUpdateWithError(e.Error) below

**Comment by angitbit** [comment]
@rmcdonald @jjoyce-bittorrent @twilmoth @janfal-bt

**Comment by jjoyce-bittorrent** [comment]
lgtm.

looks like we will be getting all of these error/notifications? that's going to be a lot of stats work
to post process this.

**Comment by angitbit** [comment]
another pr will have stats handling; do you want it to only send a generic error stat?

**Pull request merged**